poetry.lock

```diff
@@ -1,33 +1,49 @@
-# This file is automatically @generated by Poetry 1.8.2 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.8.3 and should not be changed by hand.
```

Comment:
Since this will be a dependency to other projects, I'd leave out the lock file from the repo. It doesn't need to be pinned to provide replicable results.